lib/slash-commands/ext/ext.js

```diff
@@ -41,31 +52,96 @@ class Ext {
             callback = arguments[1];
         }
 
-        Util.readJsonFromFile(FILE, (err, data) => {
-            if (err) {
-                throw err;
+        if (this._attemptedToReadSpreadsheetConfigOnce) {//if we already tried to load the config file on a previous request...
+            this._handle(searchString, callback);//handle the request
```

Comment:
This line: `this._handle(searchString, callback);` is the same for both the `if` and the `else` clause. If someday its signature changes, or if we were chasing a mysterious bug, we would be looking in two places instead of one.

Instead of this whole if/else block, why don't we simply do:
```
if (never tried to load the config file) {
    // config / json logic
}
this._handle(searchString, callback);
```
??

It will reduce the method's complexity, and reduce the area to be test covered ;)